tests/Action/AppendFormFieldElementActionTest.php

```diff
@@ -22,6 +22,7 @@
 use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
 use Sonata\AdminBundle\Admin\Pool;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
+use stdClass;
```

Comment:
AFAIK we don't use import for internal classes

Reply:
Ok, I saw it in https://github.com/sonata-project/SonataSeoBundle/pull/358/files#diff-dbdd1855f86b07a0c76f12457036ab73R19, that's why I thought it was ok